AferoSwiftSDK.podspec

```diff
@@ -2,7 +2,7 @@ Pod::Spec.new do |s|
 
     s.name         = "AferoSwiftSDK"
     s.module_name  = "Afero"
-    s.version      = "1.4.2"
+    s.version      = "1.4.3"
```

Comment:
Given the type name changes in ReactiveSwift (specifically `NoError` to `Never`), it probably makes sense to bump this to `1.5.0` as users will have to make changes internally.

Reply:
Good point